pkg/asset/ignition/templates/bootkube_sh.go

```diff
@@ -0,0 +1,109 @@
+package templates
+
+const (
+	// BootkubeShFileContents is a script file for running bootkube on the
+	// bootstrap nodes.
+	BootkubeShFileContents = `#!/bin/bash
+set -e
+
+echo "Rendering Kubernetes core manifests..."
+
+# shellcheck disable=SC2154
```

Comment:
We aren't running shellcheck on these templates, so these comments can be removed.